api/admin/api.go

```diff
@@ -32,6 +34,7 @@ func New() (*API, error) {
 	return &API{
 		Config: *c,
 		client: http.Client{},
+		Logger: logger.New(),
```

Comment:
API constructor is used only in tests so we can't pass `Logger` from `Cloudinary` here.